tests/src/python/test_qgsserver_wfs.py

```diff
@@ -506,6 +506,12 @@ def test_getFeatureFeatureId(self):
         self.wfs_request_compare(
             "GetFeature", '1.0.0', "SRSNAME=EPSG:4326&TYPENAME=testlayer&FEATUREID=testlayer.0", 'wfs_getFeature_1_0_0_featureid_0')
 
+    def test_getFeatureFeatureId(self):
+        """Test GetFeature with featureid"""
+
+        self.wfs_request_compare(
+            "GetFeature", '1.0.0', "SRSNAME=EPSG:4326&TYPENAME=testlayer&FEATUREID=testlayer.0&PROPERTYNAME=*", 'wfs_getFeature_1_0_0_featureid_0')
```

Comment:
Can you add a test with `()` here as mentioned in the description or is this already the case with this?

Reply:
tests added